demo/roles/users/tasks/main.yml

```diff
@@ -1,6 +1,14 @@
+- name: Create hpccloud user
+  become: yes
+  become_user: root
+  user: name=hpccloud shell=/bin/bash uid=1002
+
+- name: Give hpccloud passwordless sudo
+  become: yes
+  become_user: root
+  lineinfile: "dest=/etc/sudoers state=present regexp='^hpccloud' line='hpccloud ALL=(ALL) NOPASSWD: ALL'"
+
```

Comment:
Great so I may be ok without the hpccloud user. Will clean that up and try if that does work from scratch.

Thanks!